exercises/calculator.cpp

```diff
@@ -10,3 +10,54 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+using namespace std;
+
+float operations(){
+
+  float a,b;
+  cout << "Insert the first number: ";
+  cin >> a;
+  cout << endl << "Insert second number: ";
+  cin >> b;
+
+  int op;
+  float result;
+  while(true){
+    cout << endl << "Select the operation: " << endl << "1) SUM" << endl << "2)Difference" << endl << "3)Multiplication" << endl << "4)Division" << endl;
+    cin >> op;
+    if(op <1 || op >4)
+      cerr << "Wrong choice, please select a valid operation number [1-4]" << endl;
+    else if(op == 4 && b == 0)
+      cerr << "Can't divide a number for 0, please select another operation" << endl;
+    else
+      break;
+  }
+
+  switch(op){
+    case 1:
+      cout << endl << "SUM = ";
+      result = a + b;
+      break;
+    case 2:
+      cout << endl << "Difference = ";
+      result = a - b;
+      break;
+    case 3:
+      cout << endl << "Multiplication = ";
+      result = a * b;
+      break;
+    case 4:
+      cout << endl << "Division = ";
+      result = a / b;
+      break;
+  }
```

Comment:
potresti "estrarre" questo codice in un'altra funzione

Reply:
La funzione andrebbe ad implementare lo switch e restituirebbe un float alla funzione operations?

Reply:
direi di si